engine/orchestrator/score-orchestrator-impl/src/test/java/io/cloudslang/orchestrator/repositories/SuspendedExecutionsRepositoryTest.java

```diff
@@ -173,6 +173,23 @@ public void findFinishedSuspendedExecutionsNegativeTest(){
     }
 
 
+    @Test
+    public void collectCompletedSuspendedTest(){
+
+        Map<String, String> contexts = new HashMap<>();
+        contexts.put("flowContext", "");
+
+        Execution exec = new Execution(2L, 0L, contexts);
+        SuspendedExecution suspendedExecution = new SuspendedExecution("111", "888", 5, exec);
+
+        repository.save(suspendedExecution);
+
+        List<SuspendedExecution> read = repository.collectCompletedSuspendedExecutions(new PageRequest(0, 100));
+
+        Assert.assertTrue(read != null);
+        Assert.assertTrue(read.get(0).getExecutionId().equals("111"));
```

Comment:
prefer `assertThat `or `assertEquals`. In case this failse we would see more information _actual_ and _expected_